taketwo-webapi/main.py

```diff
@@ -217,9 +222,10 @@ def analyse_text(text: Text):
 @app.put("/check")
 def check_words(text: Text):
     res = []
+    db = getDb()
     for item in db.view('_all_docs',include_docs=True):
         doc = item.doc
-        if doc["category"] == "racial slur" and doc["flagged_string"].lower() in text.content.lower():
+        if doc["category"] == "racial-slur" and doc["flagged_string"].lower() in text.content.lower():
```

Comment:
Previously, this didn't match what was being returned from the read_categories method so the marks saved through template.html weren't recognized by this method

Reply:
I can take this out if we'd rather have this fixed in a separate change